crates/solver/src/solver/single_order_solver.rs

```diff
@@ -1,4 +1,5 @@
 use crate::{
+    driver::solver_settlements::merge_settlements,
```

Comment:
Should we move this module to something like `single_order_solver::merge` or something?

Fine where it is as well.

Reply:
Seems reasonable. 👌

Reply:
I'm fine either way, just thought I should ask (because where the code was used moved drastically, I thought it might make sense to move the code). No strong opinions though.

Reply:
It definitely is a valid point.
I believe when we are eventually happy with the new driver we probably should move around a bunch of code from the old driver and solver crates so maybe that would be a good time to do it.
I will merge for now and keep that in mind.